src/include/connection.h

```diff
@@ -915,10 +915,17 @@ struct __wt_connection_impl {
 #define WT_CONN_RECOVERY_COMPLETE 0x04000000u
 #define WT_CONN_RTS_THREAD_RUN 0x08000000u
 #define WT_CONN_SALVAGE 0x10000000u
-#define WT_CONN_TIERED_FIRST_FLUSH 0x20000000u
-#define WT_CONN_WAS_BACKUP 0x40000000u
+#define WT_CONN_SHUTTING_DOWN 0x20000000u
```

Comment:
This shutting down flag is used in conn_control_point_enable and disable, given that enable and disable should be their own connection API's they won't be able to race and this flag won't be needed.